src/load_it.cpp

```diff
@@ -237,9 +238,8 @@ BOOL CSoundFile::ReadIT(const BYTE *lpStream, DWORD dwMemLength)
 	if (inspossize > MAX_INSTRUMENTS) inspossize = MAX_INSTRUMENTS;
 	inspossize <<= 2;
 	memcpy(inspos, lpStream+dwMemPos, inspossize);
-	for (UINT j=0; j < (inspossize>>2); j++)
-	{
-	       inspos[j] = bswapLE32(inspos[j]);
+	for (j=0; j < (inspossize>>2); j++) {
+	    inspos[j] = bswapLE32(inspos[j]);
```

Comment:
The loop style changes don't really match anything else in the file.

Most of the other source file changes look reasonable (can't really speak for the configury changes).

Reply:
Hmph, the loop style changes crept in from my fork which were accompanied by build/warning fixes.
I can take them back if @Konstanty insists on it.